agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -156,56 +156,6 @@ func TestCommandOverrides(t *testing.T) {
 	assert.Equal(t, 21, exitCode, fmt.Sprintf("Expected exit code of 21; got %d", exitCode))
 }
 
-func TestDockerAuth(t *testing.T) {
-	agent := RunAgent(t, &AgentOptions{
-		ExtraEnvironment: map[string]string{
-			"ECS_ENGINE_AUTH_TYPE": "dockercfg",
```

Comment:
with this migration, we essentially lose testing of these env vars?

Reply:
We are still setting the same flags with these values but in task definition instead os environment variables.